.github/ISSUE_TEMPLATE/airflow_bug_report.yml

```diff
@@ -25,7 +25,6 @@ body:
         the latest release or main to see if the issue is fixed before reporting it.
       multiple: false
       options:
-        - "2.10.0b1"
```

Comment:
Should we remove this? I think we just need it as b2

Reply:
Yeah. We are merging to v2-10-stable that's why we removed it. We will update the one in main after b2 release